docs/user-guide/quick-start.md

```diff
@@ -28,6 +28,7 @@ To expose your service to the public internet, run:
 $ kubectl expose deployment my-nginx --target-port=80 --type=LoadBalancer
 service "my-nginx" exposed
 ```
+Note: The type, LoadBalancer, is highly dependent upon the underlying platform that Kubernetes is running on. If your cloudprovider doesn't have a loadbalancer implementation (e.g. OpenStack) for Kubernetes, you can simply use the allocated [NodePort](http://kubernetes.io/docs/user-guide/services/#type-nodeport) as a rudimentary form of loadblancing across your endpoints.
```

Comment:
Suggested edit:

"Note: The `type`, LoadBalancer, is highly dependent upon the underlying platform that Kubernetes is running on. If you're using a cloud provider that doesn't have an implementation of LoadBalancer for Kubernetes (such as. OpenStack), you can simply use the allocated [NodePort](http://kubernetes.io/docs/user-guide/services/#type-nodeport) as a rudimentary form of load balancing across your endpoints."